### DISCUSSION TITLE
快速上手 | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#378] [General]
# 快速上手 | FixIt

学习在几分钟内创建一个 Hugo FixIt 站点。

https://fixit.lruihao.cn/zh-cn/docs/getting-started/quick-start/

**Comment by Lruihao** [comment]
部署到 Vercel 可参考主题文档的配置 [vercel.json](https://github.com/hugo-fixit/docs/blob/main/vercel.json) 或者 [作者博客](https://github.com/Lruihao/hugo-blog) 的部署思路。
![blog-flow](https://github.com/Lruihao/hugo-blog/raw/main/assets/images/blog-flow.png)

**Comment by i4mhmh** [comment]
作者您好，我在本地部署该框架时执行`hugo server -D --disableFastRender`会提示：
~~~markdown
Current environment is "development". The "comment system", "PWA", "CDN", "fingerprint" and "analytics" will be disabled
~~~
该如何解决？

**Reply by Lruihao** [reply]
开发模式这是正常的，如果本地部署，还是建议生产打包，然后在本地部署。

**Comment by i4mhmh** [comment]
好的，我明白了，谢谢您！

From: Cell ***@***.***>
Date: Saturday, March 15, 2025 at 10:25
To: hugo-fixit/FixIt ***@***.***>
Cc: 全世界的面我都吃一遍 ***@***.***>, Comment ***@***.***>
Subject: Re: [hugo-fixit/FixIt] 快速上手 | FixIt (Discussion #378)


开发模式这是正常的，如果本地部署，还是建议生产打包，然后在本地部署。

―

**Comment by kaishuizhuhuoyu** [comment]
太好了
